lua/ulx/modules/sh/yeet.lua

```diff
@@ -0,0 +1,38 @@
+local CATEGORY_NAME = "Fun"
+
+local MIN_STRENGTH = 6000	
+local MAX_STRENGTH = 500000	
+local DEFAULT_STRENGTH = 6000 
+
+function yeetPlayer( caller, targets, strength )
+	
+	for _, v in pairs( targets ) do 
+		
+	if not v:IsPlayer() then return end
+	if not v:Alive() then return end
+
+	if v:IsFrozen() then
+		
+	v:Freeze( false )
+					
+	end
+	
+	if v:GetMoveType() == MOVETYPE_NOCLIP then
+			
+	v:SetMoveType( MOVETYPE_WALK )
+			
+	end
+
+	v:SetVelocity( Vector( math.random( 50000 ) -20000, math.random( 50000 ) -20000, math.Clamp( strength, MIN_STRENGTH, MAX_STRENGTH ) or DEFAULT_STRENGTH ) )  
```

Comment:
Typically we like to break complicated lines out into "steps".

Something like this:
```lua
local randX = math.random( 50000 ) - 20000
local randY = math.random( 50000 ) - 20000
local velocity = Vector( randX, randY, strength )
local strength = math.Clamp( strength, MIN_STRENGTH, MAX_STRENGTH )

v:SetVelocity( velocity )
```

This makes it easier for the reader to understand what's going on, how you calculated things, and how to change it.

Side note, I see your two random calculations are the same. That's usually an indication that it could be pulled out into a helper function above this one.

And one last thing; you can set default values when you define the ULX command at the bottom of this file. It might be better to set the `DEFAULT_STRENGTH` there. That way, this function can stay nice and lean.